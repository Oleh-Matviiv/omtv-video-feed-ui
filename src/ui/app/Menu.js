import React from 'react';
import styled from 'styled-components';

const Nav = styled.nav`
  display: flex;
  justify-content: center;
  background-color: #fafafa;
  color: #b4bbc1;
  padding: 1rem;
  border-bottom: .1rem solid #efefef;
`;

const Menu = () => (
  <Nav>
    VIDEO FEED
  </Nav>
);

export default Menu;
