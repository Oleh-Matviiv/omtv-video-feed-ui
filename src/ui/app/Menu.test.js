import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import Menu from './Menu';

describe('snapshots', () => {
  it('renders correclty', () => {
    const wrapper = mount(
      <Menu />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
