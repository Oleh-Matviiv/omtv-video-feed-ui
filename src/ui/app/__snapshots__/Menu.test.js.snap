// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders correclty 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: #fafafa;
  color: #b4bbc1;
  padding: 1rem;
  border-bottom: .1rem solid #efefef;
}

<Menu>
  <styled.nav>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [
            Object {
              "className": "menu",
            },
          ],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  display: flex;
  justify-content: center;
  background-color: #fafafa;
  color: #b4bbc1;
  padding: 1rem;
  border-bottom: .1rem solid #efefef;
",
            ],
          },
          "displayName": "styled.nav",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "nav",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <nav
        className="menu c0"
      >
        VIDEO FEED
      </nav>
    </StyledComponent>
  </styled.nav>
</Menu>
`;
